src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -973,3 +973,178 @@ def swap_scale_shift(weight):
         converted_state_dict[f"transformer.{key}"] = converted_state_dict.pop(key)
 
     return converted_state_dict
+
+
+def _convert_hunyuan_video_lora_to_diffusers(original_state_dict):
+    converted_state_dict = {k: original_state_dict.pop(k) for k in list(original_state_dict.keys())}
```

Comment:
We usually add a check on `original_state_dict` if it's been emptied for converters like this. Do we wanna do that here?

https://github.com/huggingface/diffusers/blob/cd991d1e1a648cffe894405db02f34059d86809f/src/diffusers/loaders/lora_conversion_utils.py#L969